test/functional/wallet_conflicts.py

```diff
@@ -346,7 +346,6 @@ def test_descendants_with_mempool_conflicts(self):
         tx1_child = bob.signrawtransactionwithwallet(raw_tx)['hex']
         tx1_child_txid = bob.sendrawtransaction(tx1_child)
 
-        # Currently neither tx1 nor tx1_child should have any conflicts
```

Comment:
In commit "wallet: track mempool conflicts" (a650caab9f11be7f5927f9aa556bb7d2a8ebed33)

Should this comment be restored? Unclear why it is being removed here

Reply:
That was a mistake, should be fixed now.